types/schema/data/Property.ts

```diff
@@ -18,8 +18,8 @@ export interface UKProperty {
   localAuthorityDistrict: string[];
   type: PropertyType;
   boundary?: {
-    site: GeoJSON;
-    area: Area;
+    site?: GeoJSON;
+    area?: Area;
```

Comment:
This is the only change I don't quite understand sorry.

Is it not the case that a boundary is either `undefined` if they upload a location plan, or all fields are required if they draw a boundary, i.e. `{ site: GeoJSON, area: Area }`

Reply:
That was also my impression of how it should be working, but empty object fields still seem to be coming through - so rather than just omitting the `boundary` key altogether it's coming through as `boundary: {}`, which was then incorrectly expecting nested keys to exist. 

Need to properly test / fix these cases (it's more than just boundary) in the passport mapping in planx-core first then can come back to tighten these types again.

Reply:
Yep - that's totally fair and makes sense if the plan is to come back and tighten up 👍 Maybe a `todo` here is a good shout?

Reply:
Decided to revert this before merging, think I'll have it worked out in the next planx-core change :+1: